cylc/uiserver/schema.py

```diff
@@ -211,9 +211,57 @@ class Arguments:
     result = GenericScalar()
 
 
+class Clean(graphene.Mutation):
+    class Meta:
+        description = sstrip('''
+            Clean a workflow from the run directory.
+        ''')
+        resolver = partial(mutator, command='clean')
+
+    class Arguments:
+        workflows = graphene.List(WorkflowID, required=True)
+        rm = graphene.String(
```

Comment:
Tested it out with `rm` and it failed. This should be a list rather than string, I think

Reply:
Yes - according to the documentation it can be a colon-separated list of strings. I've added something to deal.